src/com/lilithsthrone/game/character/effects/StatusEffect.java

```diff
@@ -5382,8 +5382,8 @@ public String extraRemovalEffects(GameCharacter target) {
 		@Override
 		public boolean isConditionsMet(GameCharacter target) {
 			float cumAmount = target.getTotalFluidInArea(SexAreaOrifice.NIPPLE);
-			return cumAmount >= CumProduction.SEVEN_MONSTROUS.getMinimumValue()
-					&& cumAmount < CumProduction.SEVEN_MONSTROUS.getMedianValue()
+			return cumAmount >= Math.pow(((Math.cbrt(10)/182.88)*target.getHeightValue())*((target.getBreastSize().getMeasurement()/3)+1),3)
```

Comment:
This calculation is almost equal to `Math.pow(target.getHeightValue()*((target.getBreastSize().getMeasurement()/3)+1), 3) / 611644` (~0.00002% difference) and is a much simpler calculation.